nautobot/dcim/tests/test_models.py

```diff
@@ -2685,34 +2661,14 @@ def test_uniqueness_parent_module(self):
         with self.assertRaises(IntegrityError):
             module_bay.save()
 
-    def test_parent_property(self):
```

Comment:
No longer relevant/needed?

Reply:
We removed the different `device` and `parent` properties so that `ModuleBay.parent` would match all of the other components' `parent` property. So I renamed `test_device_property` to `test_parent_property`

Reply:
Sorry, to clarify I actually renamed the ModuleBay `device` property to `parent` and removed this old behavior of the `parent` property that was being tested here